acceptance/clear-acceptance-space.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+
+set -eux
+
+cf api ${CF_API_URL}
+(set +x; cf auth $CF_USERNAME $CF_PASSWORD)
+
+cf target -o ${CF_ORGANIZATION}
+if ! cf space ${CF_SPACE}; then
+  exit 0
+fi
+
+cf target -s ${CF_SPACE}
+
```

Comment:
If an app or space fails to delete, would we know about it? Probably not an issue, since the kubernetes broker delete is just a few api calls, but it could still fail.